System.IO.Abstractions/FileSystemInfoBase.cs

```diff
@@ -4,9 +4,12 @@
     [Serializable]
     public abstract class FileSystemInfoBase
     {
-        protected FileSystemInfoBase(IFileSystem fileSystem)
+        private readonly string originalPath;
+
+        protected FileSystemInfoBase(IFileSystem fileSystem, string path)
```

Comment:
To reduce public API changes and therefore downstream churn please undo the changes in this file and override `ToString` in `*Wrapper`/`Mock*` instead.

Reply:
ToString removed from FileSystemInfoBase